packages/cdk8s-plus/src/container.ts

```diff
@@ -70,18 +70,27 @@ export class EnvValue {
   }
 
   /**
-   * Create a by reading a specific key inside a secret.
+   * Create an env value by reading a specific key inside a secret.
    *
    * @param secret - The secret.
    * @param key - The key.
    * @param options - Additional options.
    */
   public static fromSecret(secret: ISecret, key: string, options: EnvValueFromSecretOptions = {}): EnvValue {
```

Comment:
I'd say we get rid of this to reduce cognitive load, users can just as easily call `fromSecretValue` themselves.

Reply:
Okay, fair enough

Reply:
fixed and added BREAKING CHANGE